src/platform/intel/cavs/include/cavs/lib/pm_runtime.h

```diff
@@ -16,12 +16,17 @@
 #ifndef __CAVS_LIB_PM_RUNTIME_H__
 #define __CAVS_LIB_PM_RUNTIME_H__
 
+#define PWRD_MASK	MASK(31, 30)
+#define PWRD_BY_HPRO	BIT(31)
+#define PWRD_BY_TPLG	BIT(30)
```

Comment:
Can you explain these pm runtime actions with a inline comment. I would think that the ```PWRD_BY_TPLG``` is powering up a core if I'm following the PR correctly ?

Reply:
added  comments and better commit description to understand this.
Both of them will power up/down a core.
But for example if you will power up a core with PWRD_BY_HPRO then you need to power down with PWRD_BY_HPRO, it will prevent from power down without PWRD_BY_HPRO, same apply to PWRD_BY_TPLG